src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/DraftApprenticeship/PostPriorLearningDataRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests.DraftApprenticeship
+{
+    public class PostPriorLearningDataRequest : IPostApiRequest
+    {
+        private readonly long _cohortId;
+        private readonly long _draftApprenticeshipId;
+        public object Data { get; set; }
```

Comment:
You could make the Data property a specific type. After all, I assume only a certain type can be posted to the endpoint with any expectation of success?